src/_P121_HMC5883L.ino

```diff
@@ -127,7 +128,7 @@ boolean Plugin_121(uint8_t function, struct EventStruct *event, String &string)
 
       float heading = atan2(s_event.magnetic.y, s_event.magnetic.x);
 
-      const float decl = PCONFIG_FLOAT(0);
+      const float decl = PCONFIG_FLOAT(1);
```

Comment:
Hmm, I expect that the declination will now always be 0? Or you have to also change line 82, above...
That's the reason TD-er mentioned to use something like `#define P121_DECLINATION PCONFIG_FLOAT(0)` (for any other settings as well) so there is no confusion in what setting variable it is actually stored.